nixos/modules/installer/sd-card/sd-image.nix

```diff
@@ -229,7 +244,7 @@ in
 
         # Populate the files intended for /boot/firmware
         mkdir firmware
-        ${config.sdImage.populateFirmwareCommands}
+        cp -r ${config.sdImage.populateFirmware}/* firmware
```

Comment:
```diff
-         mkdir firmware
-         ${config.sdImage.populateFirmwareCommands}
+         cp -vr ${config.sdImage.populateFirmware} firmware
```

We can remove the weird indirection here by directly using the path as the `firmware` directory.

Also, using `-v` helps situating issues later down the line, rather than silently doing things, but that's like, my opinion, not facts.